<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1036</width>
    <height>747</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Newton Collimation Camera</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>0</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QGroupBox" name="markerCenter">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>220</y>
         <width>281</width>
         <height>91</height>
        </rect>
       </property>
       <property name="title">
        <string>Marker Center</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>58</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>Offset Y:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>58</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>Offset X:</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="offsetX">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>30</y>
          <width>71</width>
          <height>27</height>
         </rect>
        </property>
        <property name="minimum">
         <number>-500</number>
        </property>
        <property name="maximum">
         <number>500</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>30</y>
          <width>121</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>or WSAD keys</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>60</y>
          <width>121</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>+ Shift to speed up</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="offsetY">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>60</y>
          <width>71</width>
          <height>27</height>
         </rect>
        </property>
        <property name="minimum">
         <number>-500</number>
        </property>
        <property name="maximum">
         <number>500</number>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>450</y>
         <width>281</width>
         <height>111</height>
        </rect>
       </property>
       <property name="title">
        <string>Ring Diameters</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <widget class="QLabel" name="ringValue_1">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>30</y>
          <width>51</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QSlider" name="ringSlider_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>201</width>
          <height>18</height>
         </rect>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="pageStep">
         <number>10</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLabel" name="ringValue_4">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>90</y>
          <width>51</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="ringValue_3">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>70</y>
          <width>51</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QSlider" name="ringSlider_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>201</width>
          <height>18</height>
         </rect>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="pageStep">
         <number>10</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QSlider" name="ringSlider_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>90</y>
          <width>201</width>
          <height>18</height>
         </rect>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="pageStep">
         <number>10</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLabel" name="ringValue_2">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>50</y>
          <width>51</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QSlider" name="ringSlider_1">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>201</width>
          <height>18</height>
         </rect>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="pageStep">
         <number>10</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>281</width>
         <height>201</height>
        </rect>
       </property>
       <property name="title">
        <string>Camera</string>
       </property>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>71</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>Exposure:</string>
        </property>
       </widget>
       <widget class="QLabel" name="gainValue">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>160</y>
          <width>51</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>100</string>
        </property>
       </widget>
       <widget class="QSlider" name="exposureSlider">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>110</y>
          <width>191</width>
          <height>18</height>
         </rect>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>500000</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>130</y>
          <width>71</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>Gain:</string>
        </property>
       </widget>
       <widget class="QLabel" name="exposureValue">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>110</y>
          <width>51</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>100</string>
        </property>
       </widget>
       <widget class="QSlider" name="gainSlider">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>160</y>
          <width>191</width>
          <height>18</height>
         </rect>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>500</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QComboBox" name="cameraName">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>181</width>
          <height>26</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="cameraNameRefresh">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>40</y>
          <width>71</width>
          <height>26</height>
         </rect>
        </property>
        <property name="text">
         <string>Refresh</string>
        </property>
       </widget>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>570</y>
         <width>141</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string>Save Image</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="autoCenter">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>320</y>
         <width>281</width>
         <height>121</height>
        </rect>
       </property>
       <property name="title">
        <string>Automatic Center's Finder</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <widget class="QLabel" name="autoCenterQuality">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>60</y>
          <width>51</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="autoCenterRotation">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>30</y>
          <width>51</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>61</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>Rotation:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>61</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>Quality:</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1036</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
